packages/grafana-ui/src/components/Gauge/Gauge.tsx

```diff
@@ -148,3 +161,23 @@ export class Gauge extends PureComponent<Props> {
     );
   }
 }
+
+interface GaugeAutoProps {
+  titleFontSize: number;
+  gaugeHeight: number;
+  showLabel: boolean;
+}
+
+function calculateGaugeAutoProps(width: number, height: number, title: string | undefined): GaugeAutoProps {
```

Comment:
`title?: string`

Reply:
It’s not an optional argument, but I guess for typescript that is the same